modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -243,26 +243,53 @@ definition {
 	}
 
 	@description = "LPS-178192. Verifies OFF state can persist to first login user."
-	@ignore = "Test Stub"
 	@priority = 3
 	test GuestConfigurationOffStateWillPersistForFirstLogin {
+		task ("Given create a new user") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfnA",
+				userLastName = "userlnA",
+				userScreenName = "usersnA");
+		}
 
-		//task ("Given When Then") {
+		task ("And as a guest user") {
+			SignOut.signOut();
+		}
 
-		// // Given create a new user
-		// // And as a guest user
-		// // // sign out of admin role to become a guest user role
-		// // And toggle show underline to OFF state
-		// // //click show underline to OFF state
-		// // When sign in as a new user for first time
-		// // Then configuration is persisted to first time signed in user
-		// // // assert body element does not have "c-prefers-link-underline" class present
-		// // //open accessibility menu
-		// // // assert configuration is OFF state
-		// // Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
+		task ("And toggle underline is set to ON state") {
```

Comment:
Hey @john-co, since this test checks the "OFF" state, I added a new task to turn it on first because it has to be on by default.

I saw your comment on [that PR](https://github.com/liferay-peds/liferay-portal/pull/176#:~:text=The%20test%20scenarios%20here%20assumed%20ON%20state%20was%20the%20default%20state.%20You%20can%20just%20change%20all%20OFF%20state%20to%20ON%20state.%20This%20will%20likely%20apply%20to%20all%20other%20test%20scenarios.) and couldn't fit on this one.